runtime/shared.rs

```diff
@@ -59,6 +58,10 @@ extension!(runtime,
       ext.esm_files.to_mut().push(ExtensionFileSource::new("ext:runtime_main/js/99_main.js", include_str!("./js/99_main.js")));
       ext.esm_entry_point = Some("ext:runtime_main/js/99_main.js");
     }
+    #[cfg(not(feature = "webcrypto"))]
+    {
+      ext.esm_files.to_mut().push(ExtensionFileSource::new("ext:deno_crypto/00_crypto.js", "const crypto = {}; export { crypto }"));
```

Comment:
This particular line seems a bit iffy to me. I don't think this will scale if we make more extensions optional.

Reply:
I only think 3 extensions are enough for feature flags - not too hard to maintain. We should probably have standard export layout maybe if we expand this optimization